tests/test_fields.py

```diff
@@ -0,0 +1,86 @@
+import mock
+from django import test
+from django.core.exceptions import ValidationError
+from trackmaven_django import fields
+
+VALID_CHANNEL_NAMES = (
+    ('input1', 'input1'),
+    ('input2', 'input2'),
+    ('input3', 'input3'),
+    ('input4', 'input4'),
+    ('input5', 'input5'),
+    ('input6', 'input6'),
+    ('input7', 'input7'),
+    ('input8', 'input8'),
+    ('input9', 'input9'),
+    ('input10', 'input10'),
+    ('input11', 'input11'),
+    ('input12', 'input12'),
+    ('input13', 'input13'),
+    ('input14', 'input14'),
+)
+
+LIST_OF_CHANNELS = ['input1', 'input2', 'input3', 'input4', 'input5',
+                    'input6', 'input7', 'input8', 'input9', 'input10',
+                    'input11', 'input12', 'input13', 'input14']
+
+
+class TestMultipleChoiceField(test.TestCase):
+    def test_to_python(self):
+        f = fields.MultipleChoiceField(
+                choices=VALID_CHANNEL_NAMES)
+        self.assertEqual(f.to_python('input1'), ['input1'])
+        self.assertEqual(f.to_python('input1,input2,input4'),
+                         ['input1', 'input2', 'input4'])
+
+    def test_default(self):
+        f = fields.MultipleChoiceField(
+                choices=VALID_CHANNEL_NAMES,
+                default='input1,input2,input4')
+        self.assertEqual(f.get_default(), 'input1,input2,input4')
+
+    def test_get_prep_value(self):
+        f = fields.MultipleChoiceField(
+                choices=VALID_CHANNEL_NAMES,
+                default='input1,input2,input4')
+        self.assertEqual(f.get_prep_value('input1'), 'input1')
+        self.assertEqual(f.get_prep_value(['input1', 'input2', 'input4']),
+                         'input1,input2,input4')
+
+    def test_get_prep_lookup(self):
+        f = fields.MultipleChoiceField(
+                choices=VALID_CHANNEL_NAMES,
+                default='input1,input2,input4')
+        self.assertEqual(f.get_prep_lookup('exact', 'input1'), 'input1')
+        self.assertEqual(f.get_prep_lookup('contains', 'input1'), 'input1')
```

Comment:
@softinio 

I'm not sure this assert statement accurately tests the `contains` logic.  I tried it locally changing this line to `self.assertEqual(f.get_prep_lookup('contains', 'put1'), 'input1')` and it fails.  I would expect that `input1` to contain `put1`.